src/Controls/src/Core/IndicatorView.cs

```diff
@@ -162,5 +191,17 @@ void OnCollectionChanged(object sender, NotifyCollectionChangedEventArgs e)
 			}
 			Count = count;
 		}
+
+		int GetMaximumVisible()
```

Comment:
Can this maybe be an internal extension method? I see several copies of this code.

Reply:
yap, done